keras/layers/convolutional/conv2d.py

```diff
@@ -101,11 +101,11 @@ class Conv2D(Conv):
         `channels_first`.  The ordering of the dimensions in the inputs.
         `channels_last` corresponds to inputs with shape `(batch_size, height,
         width, channels)` while `channels_first` corresponds to inputs with
-        shape `(batch_size, channels, height, width)`. It defaults to the
+        shape `(batch_size, channels, height, width)`. When unspecified, uses
```

Comment:
Change "When unspecified, " to "If left unspecified, it "

Reply:
Fixed